File: core/src/main/java/co/ericp/freewayforecast/weather/WeatherPoint.java
package co.ericp.freewayforecast.weather;

import co.ericp.freewayforecast.LatLng;

/**
 * A the weather at a specific time and place.
 */
public class WeatherPoint {
    LatLng coords;
    long time;
    double temp;
    int status;
}

File: core/src/main/java/co/ericp/freewayforecast/weather/WeatherPoint.kt
package co.ericp.freewayforecast.weather

import co.ericp.freewayforecast.LatLon

/**
 * A the weather at a specific time and place.
 */
data class WeatherPoint(
        val coords: LatLon,
        val time: Long,
        val temp: Double = 0.0,
        val status: Int = 0
)

